Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -47,15 +47,85 @@ public JiraExportManager(JiraRepository.JiraRepository jiraRep)
         {
             Dictionary<Guid, string> defectsOpeningResults = new Dictionary<Guid, string>();
             List<JiraIssueExport> defectsToExport = new List<JiraIssueExport>();
+            List<string> screenshots = new List<string>();
             foreach (KeyValuePair<Guid, Dictionary<string, string>> defectForOpening in defectsForOpening)
             {
+                string summaryValue = defectForOpening.Value.ContainsKey("Summary") ? defectForOpening.Value["Summary"] : string.Empty;
+
```

Comment:
Please split this long function into smaller modules. 
CheckIfDefectExist
CreateDefect
AttachScreenshotsToDefect